fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/WindowTransformer.java

```diff
@@ -347,10 +354,25 @@ public static WindowOperator standardize(AnalyticExpr analyticExpr) {
                 return sortKey;
             }
         });
-
         return sortedGroups;
     }
 
+    private static boolean isPrefixHyperPartitionSet(List<Object> hyperSet, List<Object> subSet) {
+        if (hyperSet.isEmpty() && subSet.isEmpty()) {
+            return true;
+        }
+
+        List<List<Object>> partitionPrefix = IntStream.rangeClosed(1, hyperSet.size())
+                .mapToObj(i -> hyperSet.subList(0, i)).collect(Collectors.toList());
+        return partitionPrefix.contains(subSet);
```

Comment:
return subSet.stream().allmatch(hyperSet::contains)?

Reply:
No, this logic needs to ensure that the subset is ordered in the hyperset.
eg: HyperSet(A，B，C) , (A，B）is true, (B，A）is false

Reply:
I think it's don't need keep order subset
eg: HyperSet(A，B，C) , (A，B）is true, (B，A）is true too